### PR TITLE
Hello-Microverse Project

### PR DESCRIPTION
**Pull request by Mucyosoda**
**This is  a Hello Microverse project**
-  Add index.html file
- Add style.css file

### CODE REVIEW

**Comment by Mucyosoda** [comment]
added different files like test.md, index.html, style.css, gitignore file and linters.yml

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```
Comment (AshakaE) [inline test.md]: - [x] Please make sure this file is properly ignored from git before making a `git push`

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Mucyosoda  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```
Comment (emmanuellekamwa) [inline .gitignore]: As said by the previous TSE the test.md file should be ignored but I am still seeing it amongst your files. Please fix that.

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse project
+
+> this is my first microverse project
+
+![screenshot](./Screenshot.png)
+
```
Comment (emmanuellekamwa) [inline README.md:8]: Your screenshot is breaking. Please also try and fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛠🛑**
Hello @Mucyosoda 👋🏽
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- You did not add a descriptive pull request made up of the title and a description of the changes you have made on your project. Please try and add that.
_Check the comments under the review._👀👇🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @Mucyosoda,** 

### **Project Approved✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### **Optional Suggestion**

- Please take the TSE notes, your PR doesn't have a descriptive PR, next project add description to your PR  see 👇
pr


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**